### PR TITLE
Vnext 110

### PR DESCRIPTION
**Pull request by rklind**

### CODE REVIEW

File: src/sdk/models/policies.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+	description?: string;
+	category?: JobCategory;
+	disabled?: boolean;
+}
+
+export interface PolicyJob extends Job {
+	policy?: PolicyJobOptions;
+}
+
+export interface PolicyJobOptions extends TransferPath {
```
Comment (smbecker) [inline src/sdk/models/policies.ts:17]: This should also include a list of `Policy` objects

File: src/sdk/models/policyStatistics.ts [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+export interface PolicyStatistic {
+	name?: string;
+	description?: string;
+	bytes?: number;
+	files?: number;
+	folders?: number;
+	versions?: number;
+	storage?: number;
+	cost_savings?: number;
```
Comment (smbecker) [inline src/sdk/models/policyStatistics.ts:11]: Remove for now.

File: src/sdk/models/policyStatistics.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+export interface PolicyStatisticsList {
+	[key: string]: PolicyStatistic;
+}
+
+export type PolicyJobStatisticsTimeline = { 
+	timestamp: number; 
+	stats: {
+		by_risk?: PolicyStatisticsList;
+		by_group?: PolicyStatisticsList;
+		by_activity?: PolicyStatisticsList;
```
Comment (smbecker) [inline src/sdk/models/policyStatistics.ts:23]: `by_activity` will not be populated for phase 1. We will need to handle that correctly from the UI.

File: src/sdk/models/policyStatistics.ts [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+	}
+}[];
+
+export interface PolicyJobStatistics extends IEntityIdentifier<string> {
+	timeline?: PolicyJobStatisticsTimeline;
+	by_risk?: PolicyStatisticsList;
+	by_group?: PolicyStatisticsList;
+	by_activity?: PolicyStatisticsList;
+	by_location_health?: PolicyStatisticsList;
```
Comment (smbecker) [inline src/sdk/models/policyStatistics.ts:32]: We won't have statistics by health.

Comment (rklind) [reply src/sdk/models/policyStatistics.ts:32]: How will the `locations` pie chart be populated in the `Policy overview`? https://www.figma.com/proto/54xvnM968kcVTAKfUTjc8X/Policy-Engine?node-id=174%3A751&scaling=min-zoom

Comment (smbecker) [reply src/sdk/models/policyStatistics.ts:30]: Ugh, I guess we will need to calculate then.

File: src/sdk/models/policies.ts [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+	category?: PolicyCategory;
+	disabled?: boolean;
+}
+
+export interface PolicyJob extends Job {
+	policy?: PolicyJobOptions;
+}
+
+export interface PolicyJobOptions extends TransferPath {
+	policy?: Policy[];
```
Comment (smbecker) [inline src/sdk/models/policies.ts:18]:
```suggestion
	policies?: Policy[];
```

File: src/sdk/models/policyCategories.ts [hunk 6]
```diff
@@ -0,0 +1,5 @@
+import { IEntityIdentifier } from './base';
+
+export interface PolicyCategory extends IEntityIdentifier<number> {
```
Comment (smbecker) [inline src/sdk/models/policyCategories.ts:3]:
```suggestion
export interface PolicyCategory extends IEntityIdentifier<string> {
```

**Pull request merged**